lib/brcobranca.rb

```diff
@@ -97,6 +97,7 @@ module Boleto
     autoload :Hsbc,          'brcobranca/boleto/hsbc'
     autoload :Bradesco,      'brcobranca/boleto/bradesco'
     autoload :Caixa,         'brcobranca/boleto/caixa'
+    autoload :Sicoob,        'brcobranca/boleto/sicoob'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.